bolt/test/binary-analysis/AArch64/gs-pauth-calls.s

```diff
@@ -0,0 +1,676 @@
+// RUN: %clang %cflags -march=armv8.3-a %s -o %t.exe
+// RUN: llvm-bolt-binary-analysis --scanners=pacret %t.exe 2>&1 | FileCheck %s
+
+// FIXME In the below test cases, LR is usually not spilled as needed, as it is
+//       not checked by BOLT.
```

Comment:
I'm wondering if this needs to be a FIXME, or simply a note to explain that the test cases are simplified/artificial?
I don't think I've got a strong opinion either way. Presumably, it'd be better for the test cases to be closer to "real-world" code, so ideally, they should have the ldp/stp spill/fill instructions?

Reply:
I agree, let's use real-world code where possible. Updated the tests, thanks!